package.json

```diff
@@ -4,35 +4,51 @@
   "private": true,
   "scripts": {
     "start": "react-scripts start",
+    "s": "set BROWSER=none&& react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
+    "deploy": "npm run build && npm run deploy:s3",
+    "deploy:s3": "serverless client deploy --no-config-change --no-policy-change --no-cors-change --no-confirm",
+    "client:deploy": "sls client deploy --no-config-change --no-policy-change --no-cors-change",
+    "client:deploy:nc": "npm run client:deploy -- --no-confirm",
+    "client:build:deploy": "npm run build && npm run client:deploy",
+    "client:build:deploy:nc": "npm run build && npm run client:deploy:nc",
+    "cloudfront:setup": "sls deploy",
+    "cloudfront:domainInfo": "sls domainInfo",
+    "cloudfront:invalidateCache": "sls invalidateCloudFrontCache",
+    "cloudfront:build:deploy": "npm run client:build:deploy && npm run cloudfront:invalidateCache",
+    "cloudfront:build:deploy:nc": "npm run client:build:deploy:nc && npm run cloudfront:invalidateCache",
+    "cloudfront:update:build:deploy": "npm run cloudfront:setup && npm run cloudfront:build:deploy",
+    "cloudfront:update:build:deploy:nc": "npm run cloudfront:setup && npm run cloudfront:build:deploy:nc",
     "eject": "react-scripts eject"
   },
   "dependencies": {
-    "axios": "^0.19.2",
-    "formik": "^2.1.5",
-    "formik-material-ui": "^2.0.1",
-    "yup": "^0.29.1",
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.9.1",
+    "@reduxjs/toolkit": "^1.2.5",
+    "@types/lodash": "^4.14.158",
     "@types/node": "^12.0.0",
     "@types/react": "^16.9.43",
     "@types/react-dom": "^16.9.8",
+    "@types/react-redux": "^7.1.7",
     "@types/react-router-dom": "^5.1.5",
     "@types/yup": "^0.29.3",
+    "axios": "^0.19.2",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
+    "enzyme-to-json": "^3.4.4",
+    "formik": "^2.1.5",
+    "formik-material-ui": "^2.0.1",
+    "lodash": "^4.17.19",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-redux": "^7.2.0",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.1",
+    "serverless-finch": "^2.6.0",
```

Comment:
I would recommend installing 'serverless' also as local package instead of globally.
In this case it will be easier to setup your project with proper 'serverless' version which is completely compatible with your app. And no need to have something installed in OS but nodeJs and npm.